data/json/items/containers.json

```diff
@@ -659,7 +665,8 @@
     "color": "green",
     "contains": "10L",
     "seals": true,
-    "watertight": true
+    "watertight": true,
+    "qualities": [ [ "BOIL", 1 ] ]
```

Comment:
You added `BOIL` to "plastic jerrycan".

It would have been nice if you presented a list of touched `id`s. :(